apm-agent-plugins/apm-api-plugin/src/main/java/co/elastic/apm/agent/plugin/api/AbstractSpanInstrumentation.java

```diff
@@ -173,45 +173,45 @@ public static void getTraceId(@Advice.FieldValue(value = "span", typing = Assign
         }
     }
 
-    public static class AddTagInstrumentation extends AbstractSpanInstrumentation {
-        public AddTagInstrumentation() {
+    public static class AddStringLabelInstrumentation extends AbstractSpanInstrumentation {
+        public AddStringLabelInstrumentation() {
             super(named("doAddTag"));
         }
 
         @VisibleForAdvice
         @Advice.OnMethodEnter(suppress = Throwable.class)
-        public static void addTag(@Advice.FieldValue(value = "span", typing = Assigner.Typing.DYNAMIC) TraceContextHolder<?> context,
-                                  @Advice.Argument(0) String key, @Nullable @Advice.Argument(1) String value) {
+        public static void addLabel(@Advice.FieldValue(value = "span", typing = Assigner.Typing.DYNAMIC) TraceContextHolder<?> context,
+                                    @Advice.Argument(0) String key, @Nullable @Advice.Argument(1) String value) {
             if (value != null && context instanceof AbstractSpan) {
                 ((AbstractSpan) context).addTag(key, value);
```

Comment:
I suggest lose the tag notion internally as well, leave only OpenTracing stuff with it (because we have no choice...). This is our terminology going forward.